Geometry/HGCalCommonData/interface/HGCalDDDConstants.h

```diff
@@ -102,13 +101,16 @@ class HGCalDDDConstants {
 			  float& tl, float& alpha) const;
   bool waferInLayer(int wafer, int lay) const;
```

Comment:
I shall check this for a later version. This method was not touched in this change